Created Kakaowebtoon presence

Faelayis opened the pull request:
- Added setting buttons
- Added setting Privacy Mod
- Added setting Show timestamps

<details>
<summary>
Preview
</summary>
<br>

![image](https://user-images.githubusercontent.com/48393914/127847383-a43686e3-43f8-4e2e-83f1-80711975da90.png)![image](https://user-images.githubusercontent.com/48393914/127847405-2a167e10-7715-4b9f-8ff9-f609e72e7f9f.png)![image](https://user-images.githubusercontent.com/48393914/127847442-91615968-5f91-4d6a-94a4-eab6ba837e46.png)![image](https://user-images.githubusercontent.com/48393914/127847473-6a07df4d-615f-4e24-b601-2450900621a6.png)![image](https://user-images.githubusercontent.com/48393914/127847493-d9231163-8ce2-43ba-947c-cc90bc67e720.png)![image](https://user-images.githubusercontent.com/48393914/127847604-85df5f43-2830-4575-a209-33d65d8c0009.png)![image](https://user-images.githubusercontent.com/48393914/127847648-e236361c-3179-4ea1-8af7-41ba8bbb1bd6.png)![image](https://user-images.githubusercontent.com/48393914/127847730-5b6d49f0-d5ed-4d80-a34c-21fc30cb30db.png)![image](https://user-images.githubusercontent.com/48393914/127847767-e57fcb7a-c218-4e72-91db-e6c604f41177.png)![image](https://user-images.githubusercontent.com/48393914/127847815-2a129001-9538-40a1-a8ff-ee2ebc5b6ff1.png)![image](https://user-images.githubusercontent.com/48393914/127922920-3ed75228-989e-4953-9314-e6b6b6da1775.png)![image](https://user-images.githubusercontent.com/48393914/127922947-5416e03b-73a3-40a5-aee3-6dc017997dfd.png)


</details>

Faelayis committed:
Created Kakaowebtoon presence

Added setting buttons
Added setting Privacy Mod
Added setting Show timestamps

Faelayis committed:
Fix Kakaowebtoon

- Fixed logo and thumbnail

Bas950 reviewed (changes requested):
This should be part of a Kakao presence, not standalone.

Faelayis committed:
Update Kakao presence

- Update logo
- Change service Kakaowebtoon -> Kakao
- Added Support https://page.kakao.com
- Added Support https://store.kakaofriends.com
- Added Support https://th.kakaowebtoon.com

Faelayis committed:
Improve Buttons

Faelayis committed:
Fix does not change when changing the web

- Fixed title
- Fixed largeImageKey

SlowLife1661 commented:
> This should be part of a Kakao presence, not standalone.
